packages/preview/unito-thesis/0.1.0/lib.typ

```diff
@@ -0,0 +1,319 @@
+// Official declaration of originality, both in English and Italian
+// taken directly from Paola Gatti's email
+#let declaration-of-originality = (
+  "en": [
+    I declare to be responsible for the content I'm presenting in order to obtain the final degree, not to have plagiarized in all or part of, the work produced by others and having cited original sources in consistent way with current plagiarism regulations and copyright. I am also aware that in case of false declaration, I could incur in law penalties and my admission to final exam could be denied
+  ],
+  "it": [
+    Dichiaro di essere responsabile del contenuto dell'elaborato che presento al fine del conseguimento del titolo, di non avere plagiato in tutto o in parte il lavoro prodotto da altri e di aver citato le fonti originali in modo congruente alle normative vigenti in materia di plagio e di diritto d'autore. Sono inoltre consapevole che nel caso la mia dichiarazione risultasse mendace, potrei incorrere nelle sanzioni previste dalla legge e la mia ammissione alla prova finale potrebbe essere negata.
+  ]
+)
+
+#let template(
+  // Your thesis title
+  title: [Thesis Title],
+
+  // The academic year you're graduating in
+  academic-year: [2023/2024],
+
+  // Your thesis subtitle, should be something along the lines of
+  // "Bachelor's Thesis", "Tesi di Laurea Triennale" etc.
+  subtitle: [Bachelor's Thesis],
+
+  // The paper size, refer to https://typst.app/docs/reference/layout/page/#parameters-paper for all the available options
+  paper-size: "a4",
+
+  // Candidate's informations. You should specify a `name` key and
+  // `matricola` key
+  candidate: (),
+
+  // The thesis' supervisor (relatore)
+  supervisor: "",
+
+  // An array of the thesis' co-supervisors (correlatori).
+  // Set to `none` if not needed
+  co-supervisor: (),
+
+  // An affiliation dictionary, you should specify a `university`
+  // keyword, `school` keyword and a `degree` keyword
+  affiliation: (),
+
+  // Set to "it" for the italian template
+  lang: "en",
+
+  // The thesis' bibliography, should either be passed as a Hayagriva
+  // .yml file or a BibTeX .bib file or `none` if you don't need
+  // to include a bibliography
+  bibliography-path: none,
```

Comment:
Bibliography has the same problem as images, it should also be a `bibliography("...")` call. You can also take a look at the charged-ieee template to see how it's done there. The whole path handling will be improved in the future, but in the meantime, packages unfortunately have to work around it.